core/SConscript.unix

```diff
@@ -118,6 +118,7 @@ if EVERYTHING:
         'ENABLE_MODULE_RANGEPROOF',
         'ENABLE_MODULE_RECOVERY',
         'ENABLE_MODULE_ECDH',
+        'ENABLE_MODULE_SURJECTIONPROOF',
```

Comment:
Why we have different flag set than in `SConscript.firmware`?

Reply:
I left the range-proof & surjection-proof generation enabled on the emulator - for testing the [full blinding/unblinding API](https://github.com/trezor/trezor-firmware/blob/2601c9f18b614833322a30ddcb26aa7b457b3cfb/core/embed/extmod/modtrezorcrypto/modtrezorcrypto-secp256k1_zkp.h#L912-L951) (which is also enabled on the emulator).
We can remove the following methods from the `secp256k1_zkp.Context` - to keep the `SConscript` files the same for the device and the emulator:
```
        {MP_ROM_QSTR(MP_QSTR_rangeproof_sign),
         MP_ROM_PTR(&mod_trezorcrypto_secp256k1_context_rangeproof_sign_obj)},
        {MP_ROM_QSTR(MP_QSTR_rangeproof_rewind),
         MP_ROM_PTR(&mod_trezorcrypto_secp256k1_context_rangeproof_rewind_obj)},
        {MP_ROM_QSTR(MP_QSTR_surjection_proof),
         MP_ROM_PTR(&mod_trezorcrypto_secp256k1_context_surjection_proof_obj)},
        {MP_ROM_QSTR(MP_QSTR_verify_surjection_proof),
         MP_ROM_PTR(
             &mod_trezorcrypto_secp256k1_context_verify_surjection_proof_obj)},
```
WDYT?

Reply:
Removed unneeded code from both emulator and firmware so their SConscript files are the same (force-pushed https://github.com/trezor/trezor-firmware/pull/317/commits/16bbfe80c9d4e3ad8ec447a5271572305bb777aa).